experimental/FTheoryTools/src/LiteratureModels/Models/model1408_4808-12.json

```diff
@@ -0,0 +1,70 @@
+{
+    "model_index": "",
+    "arxiv_data": {
+        "id": "1408.4808v2",
+        "doi": "10.48550/arXiv.1408.4808",
+        "version": "2",
+        "discipline": "hep-th",
+        "link": "https://arxiv.org/abs/1408.4808v2",
+        "model_location": {
+            "section": "3",
+            "equation": "3.155",
+            "page": "70"
+        }
+    },
+    "journal_data": {
+        "doi": "10.1007/JHEP01(2015)142",
+        "report_numbers": ["UPR-1264-T", "CERN-PH-TH-2014-171", "BONN-TH-2014-13"],
+        "journal": "JHEP",
+        "volume": "01",
+        "pages": "142",
+        "year": "2015",
+        "link": "https://link.springer.com/article/10.1007/JHEP01(2015)142",
+        "model_location": {
+            "section": "3",
+            "equation": "3.154",
+            "page": "70"
+        }
+    },
+    "paper_metadata": {
+        "authors": [
+            "Denis Klevers",
+            "Damián Kaloni Mayorga Peña",
+            "Paul-Konstantin Oehlmann",
+            "Hernan Piragua",
+            "Jonas Reuter"
+        ],
+        "title": "F-Theory on all Toric Hypersurface Fibrations and its Higgs Branches",
+        "description": "All Calabi-Yau manifolds with a genus-one fiber associated to the 16 reflexive 2D polyhedra F_i",
+        "buzzwords": ["Mordell-Weil", "reflexive polyhedra", "toric varieties", "elliptic fibrations"]
+    },
+    "model_descriptors": {
+        "type": "hypersurface",
+        "description": "The toric variety associated to F_12",
+        "gauge_algebra": ["su2","su2","u1","u1"],
+        "global_gauge_quotients": [["Z4", [1, 2, 3, 4]]]
```

Comment:
Based on Fig. 21 and Table 22, it seems this should actually be `"global_gauge_quotients": [["Z2", [1, 3]], ["Z2", [2, 4]]]`, and the same is true for the WSF version